core/templates/dev/head/pages/exploration_editor/exploration_metadata_modal_directive.html

```diff
@@ -6,11 +6,10 @@
   <p>
     Just a few more things before you publish:
   </p>
-  <p ng-if="requireTitleToBeSpecified">
-    Add a title: what should your exploration be called?
-    <input type="text" class="form-control" ng-model="explorationTitleService.displayed"
-           placeholder="Music in the 19th Century" maxlength="40">
-  </p>
+  <title-field
```

Comment:
I am still not convinced. I don't think showing creator what he/she has already filled in will make the process any more complex. Creator has to just fill in empty fields. Right?